ui/src/app/ui-banner/ui-banner.tsx

```diff
@@ -52,18 +51,18 @@ export class Banner extends React.Component {
                                 </span>
                             </button>
                             <div className='ui_banner__text'>
-                                <div>
-                                    <a href={apiData.announcements['url']} className='ui_banner__items'>
+                                <div className='ui_banner__items'>
+                                    <a href={apiData.announcements['url']} target='_blank' className='ui_banner__items'>
```

Comment:
Hi @gaganhegde , could you try adding `rel="external"` to these three places to try to get rid of the code scan warning?

Reply:
Also run yarn lint from the ui folder to see the linting errors that need to be fixed